js/ftx.js

```diff
@@ -478,30 +482,70 @@ module.exports = class ftx extends Exchange {
             const id = this.safeString (market, 'name');
             const baseId = this.safeString2 (market, 'baseCurrency', 'underlying');
             const quoteId = this.safeString (market, 'quoteCurrency', 'USD');
-            const type = this.safeString (market, 'type');
             const base = this.safeCurrencyCode (baseId);
             const quote = this.safeCurrencyCode (quoteId);
+            const isFuture = this.safeString (market, 'type') === 'future';
+            const contract = isFuture;
+            const name = this.safeString (market, 'name');
+            let symbol = base + '/' + quote;
+            const splitName = name.split ('-');
+            const settleId = contract ? 'USD' : undefined;
+            const spot = !contract;
+            const margin = !contract;
+            const swap = (this.safeString (splitName, 1) === 'PERP');
+            const option = false;
+            let futures = false;
+            let type = 'spot';
+            let expiry = undefined;
+            if (swap) {
+                type = 'swap';
+                symbol = base + '/' + quote + ':' + settleId;
+            } else if (contract) {
+                type = 'futures';
```

Comment:
let's not change future -> futures for the sake of it

Reply:
> let's not change future -> futures for the sake of it

It should be unified across exchanges